### PR TITLE
Add technical mips best practices

### PR DESCRIPTION
**Pull request by patrick-j-govalpha**
Staging - [1](https://longforwisdom.gitbook.io/new-collection/governance/technical-mips/technical-mips-development-best-practices), [2](https://longforwisdom.gitbook.io/new-collection/governance/technical-mips/technical-mip-full-example)

### CODE REVIEW

File: SUMMARY.md [hunk 1]
```diff
@@ -6,40 +6,37 @@
...
 * Verification
   * [Executive Audit](governance/executive-audit.md)
-
 * Offchain Incentives
   * [Community SourceCred](governance/sourcecred.md)
   * [Delegate SourceCred](governance/delegate-sourcecred.md)
   * [Proposal Bounties](governance/proposal-bounties.md)
+* Technical MIPs
+  * [Technical MIPs Development Best Practices](governance/technical-mips-development-best-practices.md)
+  * [Full Example](governance/technical-mip-full-example.md)
```
Comment (LongForWisdom) [inline SUMMARY.md:39]: This sounds like it would be a full example of a technical MIP, which is not the case.

File: SUMMARY.md [hunk 2]
```diff
@@ -6,40 +6,37 @@
...
-
 * Verification
   * [Executive Audit](governance/executive-audit.md)
-
 * Offchain Incentives
   * [Community SourceCred](governance/sourcecred.md)
   * [Delegate SourceCred](governance/delegate-sourcecred.md)
   * [Proposal Bounties](governance/proposal-bounties.md)
+* Technical MIPs
+  * [Technical MIPs Development Best Practices](governance/technical-mips-development-best-practices.md)
```
Comment (LongForWisdom) [inline SUMMARY.md:38]: Maybe just change the title to 'Best Practices' ?

File: governance/technical-mips-development-best-practices.md [hunk 3]
```diff
@@ -0,0 +1,340 @@
...
+        
+        // Do something that depends on the system being live
+    }
+
+}
+```
+
+You may ask which are the "permanent contracts". In general you should assume every contract is **not** permanent and will be replaced at some point. There are 3 exceptions to this rule:
+
+ * vat
```
Comment (LongForWisdom) [inline governance/technical-mips-development-best-practices.md:302]: Wonder if these should reference the chainlog

**Review by LongForWisdom — CHANGES_REQUESTED** [review]

File: governance/technical-mips-development-best-practices.md (hunk 3 above)
Comment (patrick-j-govalpha) [reply governance/technical-mips-development-best-practices.md:302]: how would you reference the chainlog?

**Pull request merged**